### PR TITLE
Readme in english

### PR DESCRIPTION
**Pull request by nicodh**

### CODE REVIEW

**Comment by Dignifiedquire** [comment]
Maybe we should have both?

**Comment by nicodh** [comment]
Hmm, I think the only purpose of that file is to show some info on github. Is there a multi language support for that?

**Comment by Dignifiedquire** [comment]
Not directly from github, but we could have a second file `README_DE.md` and have a link at the top of the readme to it

File: README.md [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
-* Technische Projektmanager
+* Software Developer
+* Web-Developer: Front-End and Back-End
+* Technical project manager
 
 
 ### Location
-[Freiraum](http://freiraumfreiburg.de/) – nähe Bahnhof (siehe
-[Karte](map.geojson))
+[Grünhof](http://www.gruenhof.org/) – near main station
```
Comment (rupl) [inline README.md]: "central station" is the most natural way to say "Hauptbahnhof" in english.

**Comment by stmllr** [comment]
I guess people in our business understand english.

**Pull request merged**